rotkehlchen/data_import/importers/binance.py

```diff
@@ -338,7 +352,81 @@ def process_entry(
         )
-        importer.add_ledger_action(cursor, ledger_action)
+        importer.add_ledger_action(write_cursor, ledger_action)
+
+
+class BiananceEarnProgram(BinanceSingleEntry):
+    AVAILABLE_OPERATIONS = (
+        'Simple Earn Flexible Interest',
+        'Simple Earn Flexible Subscription',
+        'Simple Earn Locked Rewards',
+        'Simple Earn Locked Redemption',
+        'Simple Earn Flexible Redemption',
+        'BNB Vault Rewards',
+        'Swap Farming Rewards',
+    )
+
+    def process_entry(
+            self,
+            write_cursor: DBCursor,
+            importer: BaseExchangeImporter,
+            timestamp: Timestamp,
+            data: BinanceCsvRow,
+    ) -> None:
+        asset = data['Coin']
+        amount = abs(data['Change'])
+        event_identifier = f'{EVENT_IDENTIFIER}_{hash_csv_row(data)}'
+        timestamp_ms = ts_sec_to_ms(timestamp)
+        staking_event = None
+        if data['Operation'] in (
+            'Simple Earn Flexible Interest',
+            'Simple Earn Locked Rewards',
+            'BNB Vault Rewards',
+            'Swap Farming Rewards',
+        ):
+            staking_event = HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=0,
+                timestamp=timestamp_ms,
+                location=Location.BINANCE,
+                event_type=HistoryEventType.STAKING,
+                event_subtype=HistoryEventSubType.REWARD,
+                asset=asset,
+                balance=Balance(amount=amount),
+                location_label='CSV import',
+                notes=f'Reward from {data["Operation"]}',
+            )
+        elif data['Operation'] == 'Simple Earn Flexible Subscription':
+            staking_event = HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=0,
+                timestamp=timestamp_ms,
+                location=Location.BINANCE,
+                event_type=HistoryEventType.STAKING,
+                event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+                asset=asset,
+                balance=Balance(amount=amount),
+                location_label='CSV import',
+                notes=f'Deposit in  {data["Operation"]}',
+            )
+        elif data['Operation'] == (
+            'Simple Earn Locked Redemption',
+            'Simple Earn Flexible Redemption',
+        ):
+            staking_event = HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=0,
+                timestamp=timestamp_ms,
+                location=Location.BINANCE,
+                event_type=HistoryEventType.STAKING,
+                event_subtype=HistoryEventSubType.REMOVE_ASSET,
+                asset=asset,
+                balance=Balance(amount=amount),
+                location_label='CSV import',
+                notes=f'Unstake {asset} from binance in {data["Operation"]}',
```

Comment:
Either say "from binance" everywhere or nowhere. I assume since `Location.BINANCE` is there then there is no need to mention it.

```suggestion
                notes=f'Unstake {asset} in {data["Operation"]}',
```